README.md
*Taken from http://www.busybox.net/*:
### BusyBox: The Swiss Army Knife of Embedded Linux

BusyBox combines tiny versions of many common UNIX utilities into a single small executable.
It provides replacements for most of the utilities you usually find in GNU fileutils, shellutils, etc.
The utilities in BusyBox generally have fewer options than their full-featured GNU cousins;
however, the options that are included provide the expected functionality and behave very much
like their GNU counterparts. BusyBox provides a fairly complete environment for any small or embedded system.

BusyBox has been written with size-optimization and limited resources in mind. It is also extremely
modular so you can easily include or exclude commands (or features) at compile time.
This makes it easy to customize your embedded systems. To create a working system, just add
some device nodes in /dev, a few configuration files in /etc, and a Linux kernel.

BusyBox is maintained by [Denys Vlasenko] (vda.linux@googlemail.com), and licensed under the
[GNU GENERAL PUBLIC LICENSE] (http://www.busybox.net/license.html) version 2. (Can also be found in this projects root folder)

### Who are the Busybox developers?

The following login accounts currently exist on busybox.net.
(I.E. these people can commit patches into the Busybox, uClibc, and buildroot projects.)
```
aldot     :Bernhard Reutner-Fischer
andersen  :Erik Andersen      - uClibc and BuildRoot maintainer
bug1      :Glenn McGrath
davidm    :David McCullough
gkajmowi  :Garrett Kajmowicz  - uClibc++ maintainer
jbglaw    :Jan-Benedict Glaw
jocke     :Joakim Tjernlund
landley   :Rob Landley
lethal    :Paul Mundt
mjn3      :Manuel Novoa III
osuadmin  :osuadmin
pgf       :Paul Fox
pkj       :Peter Kjellerstedt
prpplague :David Anders
psm       :Peter S. Mazinger
russ      :Russ Dill
sandman   :Robert Griebl
sjhill    :Steven J. Hill
solar     :Ned Ludd
timr      :Tim Riker
tobiasa   :Tobias Anderberg
vapier    :Mike Frysinger
vda       :Denys Vlasenko     - Busybox maintainer
```
##### Cross-Compilers taken from Aboriginal Linux: http://landley.net/aboriginal/ GitHub: https://github.com/landley/aboriginal

## BonBons Busybox Installer

##### These are my builds of the busybox build for ARM, ARM64, x86 and MIPS devices, designed to be used on an android device

### **Why use these builds?**
##### Compared to other busybox builds, I have few additional applets added and other obsolete ones removed.
##### There's also some speed and size tweaks in place. But the main improvement, is the three busybox versions:

**A Full build**, that has all the applets any user could ever dream of.
Android ROM and OS creators could benefit from including the full build to
their packages as it has every single applet a system might need.
(Not suggested for regular users as it has many useless applets)

**A modular build**, that is suited for everyday users. It's just what you need if someone
or something needs busybox installed. It has everything needed for regular android use

**A minimalistic build**, that only has core applets often needed for scripting.
**It's ideal for mod makers or other script creators to include the minimalistic build
inside their work, for ensured compatibility and speed / size optimizations!

## More info and proper download links can be found on this projects XDA thread, [here] (http://forum.xda-developers.com/android/software-hacking/busybox-bonbon-s-busybox-t3253400)

## Build info and other documents in /docs/README.md


CHANGELOG
03/06/2016
- Updated source to stable 1.24.2
- Added Unstable 1.25.0 source
- Removed and added applets to full, modular and minimalistic builds
- (Again) Tweaked the build config to save space and to improve speed
- Fixed few bugs
- Updated compilers for (possibly) smaller binaries
- Added builds for x86_64 and mips64
- Renamed few files for easier building and installing
- Added smart install
- Added OTA survival
- Added a file containing the changelog
- Added a builder script for faster building
- Added a new build: Snapshot. This is the experiemental version of busybox
  with all the latest fixes, changes and bugs. It will be build with modular
  configuration.
- Added a dynamic installer script (Still in testing, no download link will be provided)
- Updated old installers
13/12/2015
- Removed applet: Wall, as it doen't work on android
- Modified build to fix few bugs on older devices
- Modified build to add speed and to reduce size
- Modified build to better suit the different busybox builds
- Moved back to previous way of removing old busybox symlinks, using find, to more completely delete them
25/11/2015
- Some minor fixes to installer
- Some tweaks to text appearance
- Made previous busybox version shown during installation
23/11/2015
- Some minor fixes to the installer
22/11/2015
- Added full build
- Added Minimal build
- Removed few applets from modular build as they were obsolete / breaking things
- Added few applets and features to modular build
20/11/2015
- Initial build


Busybox_Sources/Stable/include/applet_tables.h
/* This is a generated file, don't edit */

#define NUM_APPLETS 272

const char applet_names[] ALIGN1 = ""
"[" "\0"
"[[" "\0"
"acpid" "\0"
"adjtimex" "\0"
"arp" "\0"
"arping" "\0"
"ash" "\0"
"awk" "\0"
"basename" "\0"
"bash" "\0"
"blkid" "\0"
"blockdev" "\0"
"bootchartd" "\0"
"brctl" "\0"
"bunzip2" "\0"
"bzcat" "\0"
"bzip2" "\0"
"cal" "\0"
"cat" "\0"
"catv" "\0"
"chattr" "\0"
"chgrp" "\0"
"chmod" "\0"
"chown" "\0"
"chpst" "\0"
"chrt" "\0"
"cksum" "\0"
"clear" "\0"
"cmp" "\0"
"comm" "\0"
"conspy" "\0"
"cp" "\0"
"cpio" "\0"
"crond" "\0"
"crontab" "\0"
"cttyhack" "\0"
"cut" "\0"
"date" "\0"
"dc" "\0"
"dd" "\0"
"depmod" "\0"
"df" "\0"
"diff" "\0"
"dirname" "\0"
"dmesg" "\0"
"dnsd" "\0"
"dnsdomainname" "\0"
"dos2unix" "\0"
"du" "\0"
"echo" "\0"
"egrep" "\0"
"env" "\0"
"envdir" "\0"
"ether-wake" "\0"
"expand" "\0"
"expr" "\0"
"fakeidentd" "\0"
"false" "\0"
"fatattr" "\0"
"fbset" "\0"
"fbsplash" "\0"
"fdisk" "\0"
"fgrep" "\0"
"find" "\0"
"findfs" "\0"
"flash_eraseall" "\0"
"flash_lock" "\0"
"flash_unlock" "\0"
"flashcp" "\0"
"fold" "\0"
"free" "\0"
"fsck" "\0"
"fsck.minix" "\0"
"fstrim" "\0"
"fsync" "\0"
"ftpd" "\0"
"ftpget" "\0"
"ftpput" "\0"
"fuser" "\0"
"getopt" "\0"
"getty" "\0"
"grep" "\0"
"groups" "\0"
"gunzip" "\0"
"gzip" "\0"
"halt" "\0"
"hd" "\0"
"head" "\0"
"hexdump" "\0"
"hostname" "\0"
"httpd" "\0"
"hush" "\0"
"hwclock" "\0"
"id" "\0"
"ifconfig" "\0"
"ifenslave" "\0"
"ifplugd" "\0"
"init" "\0"
"inotifyd" "\0"
"insmod" "\0"
"install" "\0"
"ionice" "\0"
"iostat" "\0"
"ipcalc" "\0"
"ipcrm" "\0"
"ipcs" "\0"
"kbd_mode" "\0"
"kill" "\0"
"killall" "\0"
"killall5" "\0"
"klogd" "\0"
"less" "\0"
"linuxrc" "\0"
"ln" "\0"
"logger" "\0"
"login" "\0"
"logread" "\0"
"losetup" "\0"
"ls" "\0"
"lsattr" "\0"
"lsmod" "\0"
"lsof" "\0"
"lspci" "\0"
"lsusb" "\0"
"lzop" "\0"
"lzopcat" "\0"
"makedevs" "\0"
"md5sum" "\0"
"mdev" "\0"
"mesg" "\0"
"microcom" "\0"
"mkdir" "\0"
"mkdosfs" "\0"
"mkfifo" "\0"
"mkfs.minix" "\0"
"mkfs.vfat" "\0"
"mknod" "\0"
"mkswap" "\0"
"mktemp" "\0"
"modinfo" "\0"
"modprobe" "\0"
"more" "\0"
"mount" "\0"
"mountpoint" "\0"
"mpstat" "\0"
"msh" "\0"
"mv" "\0"
"nameif" "\0"
"nanddump" "\0"
"nandwrite" "\0"
"nbd-client" "\0"
"nc" "\0"
"netstat" "\0"
"nice" "\0"
"nmeter" "\0"
"nohup" "\0"
"nslookup" "\0"
"ntpd" "\0"
"patch" "\0"
"pgrep" "\0"
"pidof" "\0"
"ping" "\0"
"ping6" "\0"
"pipe_progress" "\0"
"pivot_root" "\0"
"pkill" "\0"
"pmap" "\0"
"poweroff" "\0"
"printenv" "\0"
"printf" "\0"
"ps" "\0"
"pscan" "\0"
"pstree" "\0"
"pwd" "\0"
"pwdx" "\0"
"rdate" "\0"
"rdev" "\0"
"readahead" "\0"
"readlink" "\0"
"readprofile" "\0"
"realpath" "\0"
"reboot" "\0"
"renice" "\0"
"reset" "\0"
"resize" "\0"
"rev" "\0"
"rfkill" "\0"
"rm" "\0"
"rmdir" "\0"
"rmmod" "\0"
"route" "\0"
"rpm" "\0"
"rpm2cpio" "\0"
"rtcwake" "\0"
"run-parts" "\0"
"rx" "\0"
"script" "\0"
"scriptreplay" "\0"
"sed" "\0"
"seq" "\0"
"setserial" "\0"
"setsid" "\0"
"setuidgid" "\0"
"sh" "\0"
"sha1sum" "\0"
"sha256sum" "\0"
"sha3sum" "\0"
"sha512sum" "\0"
"showkey" "\0"
"slattach" "\0"
"sleep" "\0"
"smemcap" "\0"
"softlimit" "\0"
"sort" "\0"
"split" "\0"
"start-stop-daemon" "\0"
"stat" "\0"
"strings" "\0"
"stty" "\0"
"su" "\0"
"sulogin" "\0"
"sum" "\0"
"swapoff" "\0"
"swapon" "\0"
"switch_root" "\0"
"sync" "\0"
"sysctl" "\0"
"syslogd" "\0"
"tac" "\0"
"tail" "\0"
"tar" "\0"
"taskset" "\0"
"tee" "\0"
"telnet" "\0"
"test" "\0"
"tftp" "\0"
"tftpd" "\0"
"time" "\0"
"timeout" "\0"
"top" "\0"
"touch" "\0"
"tr" "\0"
"traceroute" "\0"
"traceroute6" "\0"
"true" "\0"
"truncate" "\0"
"tty" "\0"
"ttysize" "\0"
"tune2fs" "\0"
"uevent" "\0"
"umount" "\0"
"uname" "\0"
"unexpand" "\0"
"uniq" "\0"
"unix2dos" "\0"
"unlink" "\0"
"unlzop" "\0"
"unzip" "\0"
"uptime" "\0"
"usleep" "\0"
"uudecode" "\0"
"uuencode" "\0"
"vi" "\0"
"watch" "\0"
"watchdog" "\0"
"wc" "\0"
"wget" "\0"
"which" "\0"
"xargs" "\0"
"yes" "\0"
"zcat" "\0"
"zcip" "\0"
;

#define APPLET_NO_acpid 2
#define APPLET_NO_adjtimex 3
#define APPLET_NO_arp 4
#define APPLET_NO_arping 5
#define APPLET_NO_ash 6
#define APPLET_NO_awk 7
#define APPLET_NO_basename 8
#define APPLET_NO_bash 9
#define APPLET_NO_blkid 10
#define APPLET_NO_blockdev 11
#define APPLET_NO_bootchartd 12
#define APPLET_NO_brctl 13
#define APPLET_NO_bunzip2 14
#define APPLET_NO_bzcat 15
#define APPLET_NO_bzip2 16
#define APPLET_NO_cal 17
#define APPLET_NO_cat 18
#define APPLET_NO_catv 19
#define APPLET_NO_chattr 20
#define APPLET_NO_chgrp 21
#define APPLET_NO_chmod 22
#define APPLET_NO_chown 23
#define APPLET_NO_chpst 24
#define APPLET_NO_chrt 25
#define APPLET_NO_cksum 26
#define APPLET_NO_clear 27
#define APPLET_NO_cmp 28
#define APPLET_NO_comm 29
#define APPLET_NO_conspy 30
#define APPLET_NO_cp 31
#define APPLET_NO_cpio 32
#define APPLET_NO_crond 33
#define APPLET_NO_crontab 34
#define APPLET_NO_cttyhack 35
#define APPLET_NO_cut 36
#define APPLET_NO_date 37
#define APPLET_NO_dc 38
#define APPLET_NO_dd 39
#define APPLET_NO_depmod 40
#define APPLET_NO_df 41
#define APPLET_NO_diff 42
#define APPLET_NO_dirname 43
#define APPLET_NO_dmesg 44
#define APPLET_NO_dnsd 45
#define APPLET_NO_dnsdomainname 46
#define APPLET_NO_dos2unix 47
#define APPLET_NO_du 48
#define APPLET_NO_echo 49
#define APPLET_NO_egrep 50
#define APPLET_NO_env 51
#define APPLET_NO_envdir 52
#define APPLET_NO_expand 54
#define APPLET_NO_expr 55
#define APPLET_NO_fakeidentd 56
#define APPLET_NO_false 57
#define APPLET_NO_fatattr 58
#define APPLET_NO_fbset 59
#define APPLET_NO_fbsplash 60
#define APPLET_NO_fdisk 61
#define APPLET_NO_fgrep 62
#define APPLET_NO_find 63
#define APPLET_NO_findfs 64
#define APPLET_NO_flash_eraseall 65
#define APPLET_NO_flash_lock 66
#define APPLET_NO_flash_unlock 67
#define APPLET_NO_flashcp 68
#define APPLET_NO_fold 69
#define APPLET_NO_free 70
#define APPLET_NO_fsck 71
#define APPLET_NO_fstrim 73
#define APPLET_NO_fsync 74
#define APPLET_NO_ftpd 75
#define APPLET_NO_ftpget 76
#define APPLET_NO_ftpput 77
#define APPLET_NO_fuser 78
#define APPLET_NO_getopt 79
#define APPLET_NO_getty 80
#define APPLET_NO_grep 81
#define APPLET_NO_groups 82
#define APPLET_NO_gunzip 83
#define APPLET_NO_gzip 84
#define APPLET_NO_halt 85
#define APPLET_NO_hd 86
#define APPLET_NO_head 87
#define APPLET_NO_hexdump 88
#define APPLET_NO_hostname 89
#define APPLET_NO_httpd 90
#define APPLET_NO_hush 91
#define APPLET_NO_hwclock 92
#define APPLET_NO_id 93
#define APPLET_NO_ifconfig 94
#define APPLET_NO_ifenslave 95
#define APPLET_NO_ifplugd 96
#define APPLET_NO_init 97
#define APPLET_NO_inotifyd 98
#define APPLET_NO_insmod 99
#define APPLET_NO_install 100
#define APPLET_NO_ionice 101
#define APPLET_NO_iostat 102
#define APPLET_NO_ipcalc 103
#define APPLET_NO_ipcrm 104
#define APPLET_NO_ipcs 105
#define APPLET_NO_kbd_mode 106
#define APPLET_NO_kill 107
#define APPLET_NO_killall 108
#define APPLET_NO_killall5 109
#define APPLET_NO_klogd 110
#define APPLET_NO_less 111
#define APPLET_NO_linuxrc 112
#define APPLET_NO_ln 113
#define APPLET_NO_logger 114
#define APPLET_NO_login 115
#define APPLET_NO_logread 116
#define APPLET_NO_losetup 117
#define APPLET_NO_ls 118
#define APPLET_NO_lsattr 119
#define APPLET_NO_lsmod 120
#define APPLET_NO_lsof 121
#define APPLET_NO_lspci 122
#define APPLET_NO_lsusb 123
#define APPLET_NO_lzop 124
#define APPLET_NO_lzopcat 125
#define APPLET_NO_makedevs 126
#define APPLET_NO_md5sum 127
#define APPLET_NO_mdev 128
#define APPLET_NO_mesg 129
#define APPLET_NO_microcom 130
#define APPLET_NO_mkdir 131
#define APPLET_NO_mkdosfs 132
#define APPLET_NO_mkfifo 133
#define APPLET_NO_mknod 136
#define APPLET_NO_mkswap 137
#define APPLET_NO_mktemp 138
#define APPLET_NO_modinfo 139
#define APPLET_NO_modprobe 140
#define APPLET_NO_more 141
#define APPLET_NO_mount 142
#define APPLET_NO_mountpoint 143
#define APPLET_NO_mpstat 144
#define APPLET_NO_msh 145
#define APPLET_NO_mv 146
#define APPLET_NO_nameif 147
#define APPLET_NO_nanddump 148
#define APPLET_NO_nandwrite 149
#define APPLET_NO_nc 151
#define APPLET_NO_netstat 152
#define APPLET_NO_nice 153
#define APPLET_NO_nmeter 154
#define APPLET_NO_nohup 155
#define APPLET_NO_nslookup 156
#define APPLET_NO_ntpd 157
#define APPLET_NO_patch 158
#define APPLET_NO_pgrep 159
#define APPLET_NO_pidof 160
#define APPLET_NO_ping 161
#define APPLET_NO_ping6 162
#define APPLET_NO_pipe_progress 163
#define APPLET_NO_pivot_root 164
#define APPLET_NO_pkill 165
#define APPLET_NO_pmap 166
#define APPLET_NO_poweroff 167
#define APPLET_NO_printenv 168
#define APPLET_NO_printf 169
#define APPLET_NO_ps 170
#define APPLET_NO_pscan 171
#define APPLET_NO_pstree 172
#define APPLET_NO_pwd 173
#define APPLET_NO_pwdx 174
#define APPLET_NO_rdate 175
#define APPLET_NO_rdev 176
#define APPLET_NO_readahead 177
#define APPLET_NO_readlink 178
#define APPLET_NO_readprofile 179
#define APPLET_NO_realpath 180
#define APPLET_NO_reboot 181
#define APPLET_NO_renice 182
#define APPLET_NO_reset 183
#define APPLET_NO_resize 184
#define APPLET_NO_rev 185
#define APPLET_NO_rfkill 186
#define APPLET_NO_rm 187
#define APPLET_NO_rmdir 188
#define APPLET_NO_rmmod 189
#define APPLET_NO_route 190
#define APPLET_NO_rpm 191
#define APPLET_NO_rpm2cpio 192
#define APPLET_NO_rtcwake 193
#define APPLET_NO_rx 195
#define APPLET_NO_script 196
#define APPLET_NO_scriptreplay 197
#define APPLET_NO_sed 198
#define APPLET_NO_seq 199
#define APPLET_NO_setserial 200
#define APPLET_NO_setsid 201
#define APPLET_NO_setuidgid 202
#define APPLET_NO_sh 203
#define APPLET_NO_sha1sum 204
#define APPLET_NO_sha256sum 205
#define APPLET_NO_sha3sum 206
#define APPLET_NO_sha512sum 207
#define APPLET_NO_showkey 208
#define APPLET_NO_slattach 209
#define APPLET_NO_sleep 210
#define APPLET_NO_smemcap 211
#define APPLET_NO_softlimit 212
#define APPLET_NO_sort 213
#define APPLET_NO_split 214
#define APPLET_NO_stat 216
#define APPLET_NO_strings 217
#define APPLET_NO_stty 218
#define APPLET_NO_su 219
#define APPLET_NO_sulogin 220
#define APPLET_NO_sum 221
#define APPLET_NO_swapoff 222
#define APPLET_NO_swapon 223
#define APPLET_NO_switch_root 224
#define APPLET_NO_sync 225
#define APPLET_NO_sysctl 226
#define APPLET_NO_syslogd 227
#define APPLET_NO_tac 228
#define APPLET_NO_tail 229
#define APPLET_NO_tar 230
#define APPLET_NO_taskset 231
#define APPLET_NO_tee 232
#define APPLET_NO_telnet 233
#define APPLET_NO_test 234
#define APPLET_NO_tftp 235
#define APPLET_NO_tftpd 236
#define APPLET_NO_time 237
#define APPLET_NO_timeout 238
#define APPLET_NO_top 239
#define APPLET_NO_touch 240
#define APPLET_NO_tr 241
#define APPLET_NO_traceroute 242
#define APPLET_NO_traceroute6 243
#define APPLET_NO_true 244
#define APPLET_NO_truncate 245
#define APPLET_NO_tty 246
#define APPLET_NO_ttysize 247
#define APPLET_NO_tune2fs 248
#define APPLET_NO_uevent 249
#define APPLET_NO_umount 250
#define APPLET_NO_uname 251
#define APPLET_NO_unexpand 252
#define APPLET_NO_uniq 253
#define APPLET_NO_unix2dos 254
#define APPLET_NO_unlink 255
#define APPLET_NO_unlzop 256
#define APPLET_NO_unzip 257
#define APPLET_NO_uptime 258
#define APPLET_NO_usleep 259
#define APPLET_NO_uudecode 260
#define APPLET_NO_uuencode 261
#define APPLET_NO_vi 262
#define APPLET_NO_watch 263
#define APPLET_NO_watchdog 264
#define APPLET_NO_wc 265
#define APPLET_NO_wget 266
#define APPLET_NO_which 267
#define APPLET_NO_xargs 268
#define APPLET_NO_yes 269
#define APPLET_NO_zcat 270
#define APPLET_NO_zcip 271

#ifndef SKIP_applet_main
int (*const applet_main[])(int argc, char **argv) = {
test_main,
test_main,
acpid_main,
adjtimex_main,
arp_main,
arping_main,
ash_main,
awk_main,
basename_main,
ash_main,
blkid_main,
blockdev_main,
bootchartd_main,
brctl_main,
bunzip2_main,
bunzip2_main,
bzip2_main,
cal_main,
cat_main,
catv_main,
chattr_main,
chgrp_main,
chmod_main,
chown_main,
chpst_main,
chrt_main,
cksum_main,
clear_main,
cmp_main,
comm_main,
conspy_main,
cp_main,
cpio_main,
crond_main,
crontab_main,
cttyhack_main,
cut_main,
date_main,
dc_main,
dd_main,
modprobe_main,
df_main,
diff_main,
dirname_main,
dmesg_main,
dnsd_main,
hostname_main,
dos2unix_main,
du_main,
echo_main,
grep_main,
env_main,
chpst_main,
ether_wake_main,
expand_main,
expr_main,
fakeidentd_main,
false_main,
fatattr_main,
fbset_main,
fbsplash_main,
fdisk_main,
grep_main,
find_main,
findfs_main,
flash_eraseall_main,
flash_lock_unlock_main,
flash_lock_unlock_main,
flashcp_main,
fold_main,
free_main,
fsck_main,
fsck_minix_main,
fstrim_main,
fsync_main,
ftpd_main,
ftpgetput_main,
ftpgetput_main,
fuser_main,
getopt_main,
getty_main,
grep_main,
id_main,
gunzip_main,
gzip_main,
halt_main,
hexdump_main,
head_main,
hexdump_main,
hostname_main,
httpd_main,
hush_main,
hwclock_main,
id_main,
ifconfig_main,
ifenslave_main,
ifplugd_main,
init_main,
inotifyd_main,
modprobe_main,
install_main,
ionice_main,
iostat_main,
ipcalc_main,
ipcrm_main,
ipcs_main,
kbd_mode_main,
kill_main,
kill_main,
kill_main,
klogd_main,
less_main,
init_main,
ln_main,
logger_main,
login_main,
logread_main,
losetup_main,
ls_main,
lsattr_main,
modprobe_main,
lsof_main,
lspci_main,
lsusb_main,
lzop_main,
lzop_main,
makedevs_main,
md5_sha1_sum_main,
mdev_main,
mesg_main,
microcom_main,
mkdir_main,
mkfs_vfat_main,
mkfifo_main,
mkfs_minix_main,
mkfs_vfat_main,
mknod_main,
mkswap_main,
mktemp_main,
modinfo_main,
modprobe_main,
more_main,
mount_main,
mountpoint_main,
mpstat_main,
msh_main,
mv_main,
nameif_main,
nandwrite_main,
nandwrite_main,
nbdclient_main,
nc_main,
netstat_main,
nice_main,
nmeter_main,
nohup_main,
nslookup_main,
ntpd_main,
patch_main,
pgrep_main,
pidof_main,
ping_main,
ping6_main,
pipe_progress_main,
pivot_root_main,
pgrep_main,
pmap_main,
halt_main,
printenv_main,
printf_main,
ps_main,
pscan_main,
pstree_main,
pwd_main,
pwdx_main,
rdate_main,
rdev_main,
readahead_main,
readlink_main,
readprofile_main,
realpath_main,
halt_main,
renice_main,
reset_main,
resize_main,
rev_main,
rfkill_main,
rm_main,
rmdir_main,
modprobe_main,
route_main,
rpm_main,
rpm2cpio_main,
rtcwake_main,
run_parts_main,
rx_main,
script_main,
scriptreplay_main,
sed_main,
seq_main,
setserial_main,
setsid_main,
chpst_main,
ash_main,
md5_sha1_sum_main,
md5_sha1_sum_main,
md5_sha1_sum_main,
md5_sha1_sum_main,
showkey_main,
slattach_main,
sleep_main,
smemcap_main,
chpst_main,
sort_main,
split_main,
start_stop_daemon_main,
stat_main,
strings_main,
stty_main,
su_main,
sulogin_main,
sum_main,
swap_on_off_main,
swap_on_off_main,
switch_root_main,
sync_main,
sysctl_main,
syslogd_main,
tac_main,
tail_main,
tar_main,
taskset_main,
tee_main,
telnet_main,
test_main,
tftp_main,
tftpd_main,
time_main,
timeout_main,
top_main,
touch_main,
tr_main,
traceroute_main,
traceroute6_main,
true_main,
truncate_main,
tty_main,
ttysize_main,
tune2fs_main,
uevent_main,
umount_main,
uname_main,
expand_main,
uniq_main,
dos2unix_main,
unlink_main,
lzop_main,
unzip_main,
uptime_main,
usleep_main,
uudecode_main,
uuencode_main,
vi_main,
watch_main,
watchdog_main,
wc_main,
wget_main,
which_main,
xargs_main,
yes_main,
gunzip_main,
zcip_main,
};
#endif

const uint16_t applet_nameofs[] ALIGN2 = {
0x3000,
0x3002,
0x0005,
0x000b,
0x0014,
0x0018,
0x001f,
0x2023,
0x3027,
0x0030,
0x0035,
0x003b,
0x0044,
0x004f,
0x0055,
0x005d,
0x0063,
0x0069,
0x306d,
0x0071,
0x0076,
0x207d,
0x2083,
0x2089,
0x008f,
0x0095,
0x209a,
0x00a0,
0x00a6,
0x00aa,
0x00af,
0x20b6,
0x00b9,
0x00be,
0x80c4,
0x00cc,
0x20d5,
0x00d9,
0x00de,
0x20e1,
0x00e4,
0x00eb,
0x00ee,
0x30f3,
0x00fb,
0x0101,
0x0106,
0x2114,
0x011d,
0x3120,
0x0125,
0x212b,
0x012f,
0x0136,
0x0141,
0x0148,
0x014d,
0x3158,
0x015e,
0x0166,
0x016c,
0x0175,
0x017b,
0x2181,
0x4186,
0x018d,
0x019c,
0x01a7,
0x01b4,
0x21bc,
0x01c1,
0x01c6,
0x01cb,
0x01d6,
0x31dd,
0x01e3,
0x01e8,
0x01ef,
0x01f6,
0x01fc,
0x0203,
0x0209,
0x220e,
0x0215,
0x021c,
0x0221,
0x2226,
0x2229,
0x222e,
0x0236,
0x023f,
0x0245,
0x024a,
0x2252,
0x0255,
0x025e,
0x0268,
0x0270,
0x0275,
0x027e,
0x0285,
0x028d,
0x0294,
0x029b,
0x02a2,
0x02a8,
0x02ad,
0x02b6,
0x02bb,
0x02c3,
0x02cc,
0x02d2,
0x02d7,
0x22df,
0x02e2,
0x82e9,
0x02ef,
0x02f7,
0x22ff,
0x0302,
0x0309,
0x030f,
0x0314,
0x031a,
0x0320,
0x0325,
0x032d,
0x2336,
0x033d,
0x0342,
0x0347,
0x3350,
0x0356,
0x235e,
0x0365,
0x0370,
0x237a,
0x0380,
0x0387,
0x038e,
0x0396,
0x039f,
0x03a4,
0x03aa,
0x03b5,
0x03bc,
0x03c0,
0x03c3,
0x03ca,
0x03d3,
0x03dd,
0x03e8,
0x03eb,
0x03f3,
0x03f8,
0x03ff,
0x0405,
0x040e,
0x0413,
0x0419,
0x041f,
0x4425,
0x442a,
0x0430,
0x043e,
0x0449,
0x044f,
0x0454,
0x345d,
0x3466,
0x046d,
0x0470,
0x0476,
0x347d,
0x0481,
0x0486,
0x048c,
0x0491,
0x049b,
0x04a4,
0x04b0,
0x04b9,
0x04c0,
0x04c7,
0x04cd,
0x04d4,
0x04d8,
0x34df,
0x34e2,
0x04e8,
0x04ee,
0x04f4,
0x04f8,
0x0501,
0x0509,
0x0513,
0x0516,
0x051d,
0x052a,
0x352e,
0x0532,
0x053c,
0x0543,
0x054d,
0x2550,
0x2558,
0x2562,
0x256a,
0x0574,
0x057c,
0x0585,
0x058b,
0x0593,
0x259d,
0x05a2,
0x05a8,
0x05ba,
0x05bf,
0x05c7,
0x85cc,
0x05cf,
0x05d7,
0x05db,
0x05e3,
0x05ea,
0x35f6,
0x05fb,
0x0602,
0x260a,
0x060e,
0x0613,
0x0617,
0x061f,
0x0623,
0x362a,
0x062f,
0x0634,
0x063a,
0x063f,
0x0647,
0x364b,
0x0651,
0x4654,
0x465f,
0x366b,
0x3670,
0x0679,
0x067d,
0x0685,
0x068d,
0x0694,
0x069b,
0x06a1,
0x06aa,
0x26af,
0x06b8,
0x06bf,
0x06c6,
0x06cc,
0x36d3,
0x06da,
0x06e3,
0x06ec,
0x06ef,
0x06f5,
0x06fe,
0x0701,
0x0706,
0x270c,
0x3712,
0x0716,
0x071b,
};

const uint8_t applet_install_loc[] ALIGN1 = {
0x11,
0x22,
0x22,
0x11,
0x11,
0x22,
0x22,
0x11,
0x11,
0x11,
0x11,
0x11,
0x11,
0x11,
0x11,
0x11,
0x21,
0x11,
0x11,
0x11,
0x12,
0x11,
0x21,
0x11,
0x11,
0x11,
0x21,
0x11,
0x12,
0x21,
0x22,
0x11,
0x22,
0x22,
0x12,
0x21,
0x22,
0x21,
0x11,
0x11,
0x12,
0x11,
0x21,
0x11,
0x11,
0x12,
0x12,
0x22,
0x22,
0x22,
0x11,
0x11,
0x11,
0x11,
0x21,
0x12,
0x10,
0x11,
0x22,
0x11,
0x12,
0x11,
0x11,
0x12,
0x12,
0x11,
0x12,
0x22,
0x21,
0x21,
0x12,
0x11,
0x11,
0x21,
0x22,
0x12,
0x11,
0x11,
0x21,
0x11,
0x11,
0x11,
0x12,
0x21,
0x11,
0x11,
0x11,
0x21,
0x22,
0x21,
0x21,
0x11,
0x11,
0x12,
0x21,
0x12,
0x21,
0x11,
0x11,
0x11,
0x11,
0x11,
0x11,
0x11,
0x21,
0x11,
0x11,
0x21,
0x11,
0x11,
0x12,
0x22,
0x12,
0x22,
0x11,
0x11,
0x11,
0x11,
0x12,
0x11,
0x11,
0x11,
0x11,
0x11,
0x22,
0x11,
0x11,
0x11,
0x11,
0x11,
0x11,
0x11,
0x12,
0x11,
0x11,
0x21,
};
